SamplesDashboard/SamplesDashboard/ClientApp/src/views/repositories/Repositories.tsx

```diff
@@ -366,7 +366,10 @@ function renderItemColumn(item: IRepositoryItem, index: number | undefined, colu
     const views = item.views;
     const url = item.url;
     const featureArea = item.featureArea;
-    const vulnerabilityAlertsCount = item.vulnerabilityAlerts.totalCount;  
+    if (item.vulnerabilityAlerts === null) {
+        return null;
+    }
```

Comment:
[nit pick] Typescript helps us do the null checks in an easier way, So that the `==null` check is not required

```suggestion
    if (!item.vulnerabilityAlerts) {
        return null;
    }
```